bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoHandler.java

```diff
@@ -71,32 +71,25 @@
             .of(THING_TYPE_SOCKET, THING_TYPE_INSIGHT, THING_TYPE_LIGHTSWITCH, THING_TYPE_MOTION)
             .collect(Collectors.toSet());
 
-    private Map<String, Boolean> subscriptionState = new HashMap<>();
+    private final Object upnpLock = new Object();
+    private final Object jobLock = new Object();
 
     private final Map<String, String> stateMap = Collections.synchronizedMap(new HashMap<>());
 
-    protected UpnpIOService service;
-    private WemoHttpCall wemoCall;
+    private final String BASICACTION = "basicevent";
+    private final String BASICEVENT = "basicevent1";
+    private final String INSIGHTACTION = "insight";
+    private final String INSIGHTEVENT = "insight1";
```

Comment:
Please make these `static` for being constants.

Reply:
Moved to WemoBindingConstants and made public statice